analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -425,6 +419,109 @@ class TernaryOperator
         true || ((b = null) == null) ? b.Value : false;                     // FN
 }
 
+class NullConditionalOperator
+{
+    void Basics(int? i)
+    {
+        _ = (i?.GetHashCode()).Value;                  // Noncompliant, ?. branches the state of i
```

Comment:
Unlike `GetHashCode`, which returns a `struct` (`int`), `GetType` returns a `class` (`Type`).
So `i?.GetType()` would not return the `Nullable<T>` which I need to call `.Value` on it and trigger S3655.

A cast of `null` to `Type` works, and just returns a `null`. 
S3655 would not trigger because the object being cast is not a `Nullable<T>`.
```cs
int? i = null;
var x = i?.GetType();
var y = (Type)x;
Console.WriteLine(y); // Just prints null
```

So I don't see how to get S3655 to be triggered on a `Nullable<T>` using `GetType`. 
Do you have any idea to achieve that?